libs/fairness/src/lib/FairnessWizard.tsx

```diff
@@ -54,6 +55,12 @@ export interface IFairnessPickerPropsV2 {
   onFairnessChange: (newKey: string) => void;
 }
 
+export interface IErrorPickerPropsV2 {
```

Comment:
Please don't use V2 since this is not relevant for the older V1 that's already been deleted.